### ISSUE TITLE
[BUG]: HTML report is not visible

### ISSUE THREAD

**Issue by Jeremias-Bertoldi** [IBMa/equal-access#1511]
### Project

cypress accessibility checker

### Browser

Chrome

### Operating system

MacOS

### Description

Hi guys, hope this bug finds you really good!
Looks like HTML report of getCompliance is not showing any content. Maybe I'm doing something wrong but until now I couldn't notice what.
I'm using cucumber as pre-processor.
Node version: v16.20.0
Cypress version: 12.0.2
cypress-accessibility-checker version: 4.0.0-rc.0

### Steps to reproduce

1. create an stepDefs files with this content:
`Then("I should see accessibilty issues on Cart page", () => {
    cy.getCompliance("A11y scan on Cart page").assertCompliance()
})`
2. create a .feature file whit this content:
`Scenario: AccessibilityChecks: SFRAT-9877 Performance should be logged in Address Book Page
    When I go to login page
    Then I should see accessibilty issues on Cart page`
3. execute `npx cypress open `command

(this two files sends you to the page to be tested so, you can use anythig you need)

- Expected result:
 . when the test finishes a result folder should be created and a `A11y scan on Cart page.html` file created with content

- Actual result:
  . the test finishes and the file is created but the content is not visible
<img width="232" alt="Captura de pantalla 2023-06-27 a la(s) 19 32 02" src="https://github.com/IBMa/equal-access/assets/100160945/b733b9bf-a29c-4f34-9f5f-229a85682988">
<img width="1440" alt="Captura de pantalla 2023-06-27 a la(s) 19 34 01" src="https://github.com/IBMa/equal-access/assets/100160945/5a9be492-9a82-4c22-98e0-bd712d42a4cb">

**Comment by tombrunet** [comment]
Anything showing in the console? There is some work that should be going on in the next release to consolidate all of our reporting code into one place so that the reports should be a lot more consistent across the tools. We've had some vacations and technical problems delay that a bit, but should be coming soon.

**Comment by tombrunet** [comment]
@Jeremias-Bertoldi Can you please try the latest release. We consolidated the report code and you may get different results.

**Comment by shunguoy** [comment]
@Jeremias-Bertoldi Can you please try the latest release, and let us know if the issue was fixed? thanx.

**Comment by Jeremias-Bertoldi** [comment]
Hi @shunguoy sorry for delay, I couldn't check HTML report yet because I'm getting another error with my configuration.

`Error: Parsing file /Users/chavo/solvd/pandora-repo-cypress/pandora_cypress/node_modules/cypress-accessibility-checker/src/lib/ACEngineManager.js: visitor[(override || node.type)] is not a function`

Do you need a new ticket for the same?

**Comment by shunguoy** [comment]
@Jeremias-Bertoldi  can you please open a new issue for the error?  please provide info for your environment so we can duplicate it. Thanx.

**Comment by Jeremias-Bertoldi** [comment]
sure

**Comment by tombrunet** [comment]
@Jeremias-Bertoldi Please try this with 3.1.59 that was pushed today. Should clean up the Cypress issues.

**Comment by tombrunet** [comment]
@Jeremias-Bertoldi If you're still getting this, please re-open.

**Comment by Jeremias-Bertoldi** [comment]
Hi @tombrunet sorry for delay, it works fine from my side. Thank you!!
